src/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import App from './App';
+
+import './styles/index.css';
+
+import { BrowserRouter as Router } from 'react-router-dom';
+
+import { Provider } from 'react-redux';
+import { store } from '../src/redux/stores';
+
+ReactDOM.render(
+    <React.StrictMode>
+        <Router>
+            <Provider store={store}>
+                    <App />
+            </Provider>            
+        </Router>
+    </React.StrictMode>,
+    document.getElementById('root')
+);
```

Comment:
Looks all good here! Nice concise index.js file, can't find anything that shouldn't be here. Good work 👍